test/scales/scaleTests.ts

```diff
@@ -141,8 +141,8 @@ describe("Scales", () => {
       var scale = new Plottable.Scale.Linear();
       scale.autoDomain();
       var d = scale.domain();
-      assert.equal(d[0], 0);
-      assert.equal(d[1], 1);
+      assert.strictEqual(d[0], 0);
+      assert.strictEqual(d[1], 1);
```

Comment:
Again, not part of this PR, but these asserts should also have a meaningful explanation as the 3rd argument